extensions/nba-game-viewer/src/hooks/useStandings.ts

```diff
@@ -34,8 +34,18 @@ const useStandings = () => {
 
     const easternStandings = getConferenceStandings(easternConference);
     const westernStandings = getConferenceStandings(westernConference);
+    const leagueStandings = getConferenceStandings({
+      name: "League",
+      abbreviation: "L",
+      standings: {
+        name: "League Standings",
+        entries: [...easternConference.standings.entries, ...westernConference.standings.entries],
+      },
+    }).sort((a: Team, b: Team) => {
+      return a?.wins !== b?.wins ? (b?.wins || 0) - (a?.wins || 0) : (a?.losses || 0) - (b?.losses || 0);
```

Comment:
e.seed give the ranking per Conf but to sort All of them, we use this algorythm:
- 1st we rank every teams by wins descreasingly
- if there's equalities, we rank concurrent by putting the ones with less losses first

![](https://media4.giphy.com/media/l41YdjI3ld5zTFzNe/200.gif)